pkg/cluster/cluster.go

```diff
@@ -58,20 +60,24 @@ func Version() (string, string, error) {
 		return condition.Version, majorMinor, nil
 	}
 
-	return "", "", errors.New("Undefined Cluster Version")
+	return "", "", fmt.Errorf("Undefined Cluster Version")
```

Comment:
I think `errors.New` was fine here, as we are not wrapping any existing error.

Reply:
done